### ISSUE TITLE
JetFire does not provide a mechanism for using client side certificates

### ISSUE THREAD

**Issue by nathanday** [acmacalister/jetfire#65]
More of a suggestion than a issue, I needed to use client side certificates, the way I got around this was to create a dictionary property on JFRWebSocket called additionalOutputStreamProperties which is then enumerated over applying the setProperty:forKey: for each entry to the output stream, adding something like this to JetFire would make it easy to handle any niche use case.

**Comment by acmacalister** [comment]
Agreed this is the feature request. I believe this is related to the same request we got on https://github.com/daltoniam/Starscream/issues/271

**Comment by vkolosovsky** [comment]
Is there any progress on this?

**Comment by jarrillaga** [comment]
@nathanday Did you achieve to use Client side certificates ? Can you share you code of WebSocket.swift ?
